requirements-dev.txt

```diff
@@ -0,0 +1,12 @@
+pytest==5.4.3
+setuptools==41.0.1
+mock==1.0.1
+pillow==5.4.1
+alabaster>=0.7,<0.8,!=0.7.5
+six
+commonmark==0.8.1
+recommonmark==0.5.0
+sphinx<2
+sphinx-rtd-theme<0.5
+readthedocs-sphinx-ext<2.2
+versioneer
```

Comment:
I'm a bit lost with the purpose of `requirements.txt` and `requirements-dev.txt`, what's the difference? And is it a good idea to pin specific `x.x.x` versions here?

Reply:
The requirements-dev.txt file has all the modules used for development and building/testing documentation. I locked the sphinx modules to a specific version so that it would do the same thing that is done on readthedocs during the sphinx build.
We can eliminate this file and I can just keep it local for me. I don't have any strong feelings either way.

Reply:
Doesn't https://github.com/Electrostatics/mmcif_pdbx/blob/master/requirements.txt have the same purpuse? It currently contains `pytest==5.4.3` and `sphinx_rtd_theme==0.5.0`, those are modules which are needed for testing and building documentation.

Reply:
Yes, I will drop the requirements-dev.txt file and just keep it locally for myself. I try to keep all my virtual envs locked down to only the things needed for them to run.
Thanks.

Reply:
Thanks again! I don't want to outlaw checking in of `requirements*.txt` files, I just like their purpose being clarified and ambiguity and duplication avoided (e.g. `sphinx-rtd-theme<0.5` vs. `sphinx_rtd_theme==0.5.0`, why are they different?).

Maybe commenting those files could help, they do support `#` comments :-)
https://pip.pypa.io/en/latest/reference/pip_install/#requirements-file-format

Reply:
I was trying to make sure that the documentation would build locally so I just mimicked what their build did, https://readthedocs.org/api/v2/build/12926518.txt, since `make html` did not work without the items in the requirements-dev.txt file. If this merge and tag don't work, I will try again and put comments in the requirements.txt file. Thanks for the pointer.